server/api.js

```diff
@@ -18,508 +18,368 @@
 const Validator = require('express-validator')
 
 // In-house dependencies
-const { helpers } = require('brave-alert-lib')
+const { helpers, googleHelpers } = require('brave-alert-lib')
 const db = require('./db/db')
 
-const validateGetClient = [Validator.header(['Authorization']).notEmpty(), Validator.param(['clientId']).notEmpty()]
-
-async function handleGetClient(req, res) {
-  const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
-
-  try {
-    if (validationErrors.isEmpty()) {
-      const client = await db.getClientWithId(req.params.clientId)
-
-      // check that the client exists
-      if (client == null) {
-        res.status(404).send({ status: 'error', message: 'Not Found' })
-
-        return
+// wrapper function for a given handler
+// this should preceed any handler when defining an Express route
+async function wrapper(req, res, next) {
+  // authorize with a Google ID Token submitted through the Authorization header of the request
+  // if the request is authorized, then the passed function will run
+  googleHelpers.paAuthorize(req, res, () => {
```

Comment:
Gotcha! I don't think I understood what you meant when you said that you were moving the `paAuthorize` into the wrapper. I think you could just add it to the routes. I'll add a comment down there to explain what I mean...